openpaas-james/apps/distributed/src/main/java/com/linagora/openpaas/james/app/DistributedServer.java

```diff
@@ -0,0 +1,34 @@
+package com.linagora.openpaas.james.app;
+
+import static org.apache.james.JamesServerMain.LOGGER;
+
+import org.apache.james.CassandraRabbitMQJamesConfiguration;
+import org.apache.james.CassandraRabbitMQJamesServerMain;
+import org.apache.james.GuiceJamesServer;
+import org.apache.james.JamesServerMain;
+
+import com.google.inject.Module;
+import com.google.inject.util.Modules;
+import com.linagora.openpaas.james.jmap.CustomMethodModule;
+
+public class DistributedServer {
+    public static final Module MODULES = Modules.combine(
+       new CustomMethodModule());
+
+    public static void main(String[] args) throws Exception {
+        CassandraRabbitMQJamesConfiguration configuration = CassandraRabbitMQJamesConfiguration.builder()
+            .useWorkingDirectoryEnvProperty()
+            .build();
+
+        LOGGER.info("Loading configuration {}", configuration.toString());
+        GuiceJamesServer server = createServer(configuration);
+
+        JamesServerMain.main(server);
+    }
+
+    public static GuiceJamesServer createServer(CassandraRabbitMQJamesConfiguration configuration) {
+
+        return CassandraRabbitMQJamesServerMain.createServer(configuration)
+            .combineWith(MODULES);
```

Comment:
could you take a look at Benoit's PR? If we are removing POP3, LMTP and Sieve you might want to review the modules injected too I guess :)